source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -208,6 +208,7 @@ export class CardDesignerSurface {
     private _persistentSelectedCardElement: CardElement;
     private _containsCarousel: boolean = false;
     private _currentCarouselPage: DesignerPeers.CarouselPagePeer;
+    private _currentPayload: any;
```

Comment:
should have a better name. For example, _arrayPayload, _multipleAdapticardsPayload, _needTemplatingPalyload....

Reply:
I definitely like these names better! Updating now